src/constants.ts

```diff
@@ -12,6 +13,8 @@ export const INPUT_CACHE = 'cache'
 export const INPUT_CHECK_FOR_UPDATES = 'check-for-updates'
 export const INPUT_NI_MUSL = 'native-image-musl'
 
+export const DISTRIBUTIONS_ONLY_MAJOR_VERSION = ['corretto']
```

Comment:
`corretto` does not make sense here, I'm ok with an empty array for compatibility with the upstream code
```suggestion
export const DISTRIBUTIONS_ONLY_MAJOR_VERSION = []
```